src/client/Client.cc

```diff
@@ -5124,63 +5124,6 @@ void Client::handle_cap_grant(MetaSession *session, Inode *in, Cap *cap, MClient
   m->put();
 }
 
-int Client::_getgrouplist(gid_t** sgids, uid_t uid, gid_t gid)
```

Comment:
Duplication?
Because this has been cleaned up in the https://github.com/ceph/ceph/pull/21132
Detail:
https://github.com/ceph/ceph/pull/21132/files#diff-fbcd8ab0dd666648c8e021177ce875d2L5127

Reply:
Yes, it's duplicate because the tip of this branch relies on some earlier commits in that branch. Hopefully it'll get sorted out properly when #21132.